### PR TITLE
Add animation to TrackPanel and TrackConfig

### PR DESCRIPTION
**Pull request by msmanoj**
- Added animation to the opening and closing of the TrackConfig, TrackPanel and Drawer

Related ticket:
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-123

### CODE REVIEW

File: src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx [hunk 1]
```diff
@@ -83,10 +78,33 @@ const TrackPanel: FunctionComponent<TrackPanelProps> = (
...
+      return 'calc(41px + 0vw)';
+    }
+    return props.trackPanelOpened
+      ? 'calc(-356px + 100vw)'
+      : 'calc(-36px + 100vw)';
+  };
+
+  useEffect(() => {
+    setTrackAnimation({
+      left: getbrowserWidth()
```
Comment (imran-ebi) [inline src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx]: Can you change `getbrowserWidth` to `getBrowserWidth`?

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx [hunk 2]
```diff
@@ -109,58 +98,56 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
...
+          </button>
+        </dd>
+      </dl>
+      <dl className="category">
+        <dd className={styles.trackLock}>
+          <button className={styles.trackLockBtn}>
+            <img src={trackLockBtn} />
+          </button>
+        </dd>
+        <dd className="trackHighlight disabled">
```
Comment (imran-ebi) [inline src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: Can we use the imported `styles` from CSS module?

File: src/ensembl/src/content/app/browser/Browser.tsx [hunk 3]
```diff
@@ -122,32 +127,55 @@ export const Browser: FunctionComponent<BrowserProps> = (
...
+      return 'calc(41px + 0vw)';
+    }
+    return props.trackPanelOpened
+      ? 'calc(-356px + 100vw)'
+      : 'calc(-36px + 100vw)';
+  };
+
+  useEffect(() => {
+    setTrackAnimation({
+      width: getbrowserWidth()
```
Comment (imran-ebi) [inline src/ensembl/src/content/app/browser/Browser.tsx]: Please change `getbrowserWidth` to `getBrowserWidth`.

File: src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx (hunk 2 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx]: Removed the classNames as they are not used anywhere.

File: src/ensembl/src/content/app/browser/Browser.tsx (hunk 3 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/browser/Browser.tsx]: That's done

File: src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx (hunk 1 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx]: That's done

**Pull request merged**